src/sentry/tagstore/v2/models/tagvalue.py

```diff
@@ -45,7 +46,7 @@ class Meta:
 
     __repr__ = sane_repr('project_id', '_key', 'value')
 
-    @property
+    @cached_property
     def key(self):
```

Comment:
Two things:

1. I don't understand how this failed but `GroupTagKey` and `GroupTagValue` `key` and `value` properties work?
2. I wonder if this could go back to just returning `self._key.key` because I added `select_related` to everything in the backend code...

Reply:
2: yes, it could, if we could guarantee that was always the case.

If that also is the case, we don't need @cached_property.